requirements-py2.txt

```
@@ -15,3 +15,4 @@ service_identity>=16.0.0
 six>=1.10.0
 Twisted>=16.0.0
 zope.interface>=4.1.3
+protego
```

```suggestion
protego>=0.1.15
```
Nitpick: Could you also please move `protego` to the section at the top of the file?)